modules/op-release-notes-1-8.adoc

```diff
@@ -12,14 +12,263 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
+// no link
+
+[id="chains-new-features-1-8_{context}"]
+=== Chains
+
+:FeatureName: Tekton Chains
 include::snippets/technology-preview.adoc[]
 
+* With this update, you can log in to a vault by using OIDC rather than a static token. This change means that Spire can generate the OIDC credential so that only trusted workloads are allowed to log in to the vault. Additionally, you can pass the vault address as a configuration value rather than inject it as an environment variable.
+// https://github.com/tektoncd/chains/pull/369
+// Brandon Mitchell @sudo-bmitch or Shubham Minglani
+
+[id="breaking-changes-1-8_{context}"]
+== Breaking changes
+
+// .Pipelines
+
+* This update changes the default metrics level of `TaskRun` and `PipelineRun` resources to the following values:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-observability
+  namespace: tekton-pipelines
+  labels:
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-pipelines
+data:
+  _example: |
+  ...
+    metrics.taskrun.level: "task"
+    metrics.taskrun.duration-type: "histogram"
+    metrics.pipelinerun.level: "pipeline"
+    metrics.pipelinerun.duration-type: "histogram"
+----
+// link:https://github.com/tektoncd/pipeline/pull/4630
+
+* With this update, if an annotation or label is present in both `Pipeline` and `PipelineRun` resources, the value in the `Run` type takes precedence. The same is true if an annotation or label is present in `Task` and `TaskRun` resources.
+// link:https://github.com/tektoncd/pipeline/pull/4638
+// Vincent Demeester
+
+* In {pipelines-title} 1.8, the previously deprecated `PipelineRun.Spec.ServiceAccountNames` field has been removed. Use the `PipelineRun.Spec.TaskRunSpecs` field instead.
+// link:https://github.com/tektoncd/pipeline/pull/4988
+
+* In {pipelines-title} 1.8, the previously deprecated `TaskRun.Status.ResourceResults.ResourceRef` field has been removed. Use the `TaskRun.Status.ResourceResults.ResourceName` field instead.
+// link:https://github.com/tektoncd/pipeline/pull/4977
+
+* In {pipelines-title} 1.8, the previously deprecated `Conditions` resource type has been removed. Remove the `Conditions` resource from `Pipeline` resource definitions that include it. Use `when` expressions in `PipelineRun` definitions instead.
+// link:https://github.com/tektoncd/pipeline/pull/4942
+
+// .Chains
+
+* For Tekton Chains, the `tekton-provenance` format has been removed in this release. Use the `in-toto` format by setting `"artifacts.taskrun.format": "in-toto"` in the `TektonChain` custom resource instead.
+// no link
+// Shubham Minglani
+
+// .{pac}
+
+* {pipelines-title} 1.7.x shipped with {pac} 0.5.x. The current update ships with {pac} 0.10.x. This change creates a new route in the `openshift-pipelines` namespace for the new controller. You must update this route in GitHub Apps or webhooks that use {pac}. To fetch the route, use the following command:
```

Comment:
just want to confirm is it okay to write version as `0.5.x` or shall we provide a exact one?

Reply:
This is the correct representation.